### DISCUSSION TITLE
[V5] Documentation: Confusing wording of lazy loading preloaded relationships

### DISCUSSION THREAD

**Discussion by fairhat** [adonisjs/core#1683] [General]
Hello,

https://preview.adonisjs.com/guides/model-relations/introduction#preloading-relationship
in the docs, it says:

__You can also lazy-load relationships for an existing model instance. For example:__
```ts
// Reference to logged in user
const user = auth.user

// Preload profile for the user
await user.preload('profile')

// Access profile
console.log(user.profile)
```

It's quite a bit confusing that the method "preload" can be used for actually "postloading" (for the lack of a better word).
It took me quite a bit of research in the docs to find out that lazy-loading relationships has to be done via "preloading".

Suggestions:
* Add a new subchapter to the documentation "lazy loading"
* Add a method that can/should only be used for lazyloading (maybe 'populate' or 'postload'?)

**Comment by thetutlage** [comment]
Agree. The semantics of the word itself is bit confusing. It was mainly used to be in-sync with the query builder `preload` method. However, in this case we can come up with a better name.

Thanks for the suggestions, but lemme also think of more appropriate name
